src/main/scala/guessbook/GuessBookUserInput.scala

```diff
@@ -0,0 +1,206 @@
+package guessbook
+
+import guesswho.GenderEnum
+
+import scala.io.StdIn
+import scala.util.{Failure, Success, Try}
+
+object GuessBookUserInput {
+  def main(args: Array[String]): Unit = {
+    val game = GuessBook
+    println("START OF GAME\n")
+
+    // Randomly select a book for each player
+//    val selectedBooks = List(game.selectBook(game.allBooks), game.selectBook(game.allBooks))
+
+    // Each player selects the book that the other one must guess
+    val selectedBookP2: Book = selectBookForOpponent(1)
+    val selectedBookP1: Book = selectBookForOpponent(2)
+    val selectedBooks = List(selectedBookP1, selectedBookP2)
+
+    val booksOnBoard = Array(game.allBooks, game.allBooks)
+    var gameOver = false
+    var guesser = 2
+
+    while (!gameOver) {
```

Comment:
is it possible to replace this with recursion, and also remove the vars by passing them through with recursion (and instantiate them initially by using default values in the method argument?

Reply:
Yes, now each player turn is a recursive call and guesser determines with board is queried and updated. gameOver is not needed anymore since the method terminates (no more recursive call) if that case is reached